server/http/helloapp/locations.go

```diff
@@ -78,9 +32,7 @@ func (h *helloapp) HandleLocations(w http.ResponseWriter, req *http.Request, use
 		&apps.ChannelHeaderIconLocation{
 			Location: apps.Location{
 				LocationType: apps.LocationChannelHeaderIcon,
-				Wish: apps.Wish{
-					URL: h.AppURL(PathWishSample),
-				},
+				Wish:         *apps.NewWish(AppID, h.AppURL(PathWishSample)),
```

Comment:
Maybe make the `Wish` field a `*Wish` so it is consistent with the rest of the data model?